Update interfaces/media-capabilities.idl

autofoolip opened the pull request:
This PR was automatically created by a bot.

Before merging, please check that any tests that depend on the updated IDL files still work.

If additional changes are needed, please manually create another PR based on this one.

See the [README](https://github.com/web-platform-tests/wpt/blob/master/interfaces/README.md) for how the IDL files in this directory are used.

<hr>

Source: https://github.com/tidoust/reffy-reports/blob/55c9ac7/whatwg/idl/media-capabilities.idl
Build: https://travis-ci.org/tidoust/reffy-reports/builds/457322567

foolip reviewed (changes requested):
This will require removing the `MediaCapabilitiesInfo` instance in the test using this IDL file.

autofoolip committed:
Update interfaces/media-capabilities.idl

Source: https://github.com/tidoust/reffy-reports/blob/55c9ac7/whatwg/idl/media-capabilities.idl
Build: https://travis-ci.org/tidoust/reffy-reports/builds/457322567

foolip commented:
After https://chromium-review.googlesource.com/c/chromium/src/+/1347001 is exported this PR will be obsoleted, or possibly only whitespace changes will remain.
